subiquity/models/subiquity.py

```diff
@@ -227,6 +234,11 @@ def render(self, syslog_identifier):
                     'content': open('/etc/machine-id').read(),
                     'permissions': 0o444,
                     },
+                'media_info': {
+                    'path': 'var/log/installer/media-info',
+                    'content': self._media_info(),
+                    'permissions': 0o644,
+                    },
```

Comment:
I know this is already merged, but I wonder if we should consider this sufficient.  The purpose of this file is to allow post-install debugging by being able to pinpoint which version of the installer was used to install the system.  But if the user refreshed the subiquity snap, this isn't complete, is it?  Should we include the information about the subiquity snap revision/version here, or in an additional file?

Reply:
Yes, it would be interesting to have the subiquity revision included in a file that apport picks up as part of making a crash report.  It is included in the pollinate data, so apport could maybe look at that? (I don't actually know where that ends up on disk)

Reply:
Let's discuss this in the standup cc @murraybd

Reply:
We still want the media-info as that describes the squashfs that is installed on target, as that's the primary purpose of the file - rather than just identifying the version of the installer used. Ubiquity too, can be upgraded live.